### PR TITLE
Dark theme

### PR DESCRIPTION
**Pull request by Ashish0804**
https://ashish0804.github.io/opencode-mait-dark-theme.github.io/index.html

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -10,6 +10,6 @@
     <style></style>
     <h1>The Open Source Software Community of MAIT</h1>
     <h1>OpenCode-MAIT</h1>
-	<p><font size=3>Oh no! Nothing here Houston! Go to <a href="/" alt="Go back to home page">home page</a>.</font></p>
+	<p><font size=5>Oh no! Nothing here Houston! Go to <a href="index.html" alt="Go back to home page">home page</a>.</font></p>
```
Comment (tm9k1) [inline 404.html]: > if it's not broken, don't fix it 

:)

File: about.html [hunk 2]
```diff
@@ -25,7 +25,7 @@ <h1>About Us</h1>
         We respect all technologies from Perl to Python.
         If you want to volunteer for OpenCode-MAIT you can join us through any of our social channels.
       </p>
-  <p align='center'><font size=4><a href="/">Back to home page</a></font></p>
+  <p align='center'><font size=4><a href="index.html">Back to home page</a></font></p>
```
Comment (tm9k1) [inline about.html]: same as above. It works with `/`.

File: assets/css/main.css [hunk 3]
```diff
@@ -7,7 +7,9 @@ html {
 }
 
 body {
-  font-family: 'PT Sans', sans-serif;
+  font-family: "Comic Sans MS", cursive, sans-serif;
```
Comment (tm9k1) [inline assets/css/main.css]: Let's leave unrelated changes out of this PR.

File: index.html [hunk 4]
```diff
@@ -11,7 +11,7 @@
 <a href="https://github.com/opencode-mait/opencode-mait.github.io" target = "_blank"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
 <body class="page-home">
     <section>
-        <img class="header-logo" alt="OpenCode-MAIT logo image" src="assets/img/oc_logo.jpg">
+        <img class="header-logo" alt="OpenCode-MAIT logo image" src="assets/img/oc_logo_t.png">
```
Comment (tm9k1) [inline index.html:14]: we already have `oc_logo.jpg`. Please remove the two new .PNGs as well if they are irrelevant.

File: index.html [hunk 5]
```diff
@@ -20,13 +20,12 @@ <h1> OpenCode-MAIT</h1>
 
     <section class='social-links'>
       <ul id='listtop'>
-        <li><a title='About Us' alt="About Us" href="about">About Us</a></li>
-
+        <li><a title='About Us' alt="About Us" href="about.html">About Us</a></li>
```
Comment (tm9k1) [inline index.html:23]: again, if it ain't broke, don't fix it. :)

File: index.html [hunk 6]
```diff
@@ -20,13 +20,12 @@ <h1> OpenCode-MAIT</h1>
...
         <li><a href="https://github.com/opencode-mait" target="_blank" rel=external alt="Github"
                title="Visit and collaborate with us on Github">Github</a>
                <ul>
-                <li><a href="MOM" rel=external title='MOMs for all formal discussions' alt="Minutes of all meetings held at OpenCode-MAIT">MOMs</a></li>
-                <li><a href="coc" rel=external title='Code of Conduct' alt="Necessary for all members to follow">Code of Conduct</a></li>
+                <li><a href="https://github.com/opencode-mait/MOM" rel=external title='MOMs for all formal discussions' alt="Minutes of all meetings held at OpenCode-MAIT">MOMs</a></li>
+                <li><a href="https://github.com/opencode-mait/coc" rel=external title='Code of Conduct' alt="Necessary for all members to follow">Code of Conduct</a></li>
```
Comment (tm9k1) [inline index.html:28]: same as above

**Review by tm9k1 — CHANGES_REQUESTED** [review]
thanks for the new PR, @Ashish0804. Please address the minor issues quickly as I see you have worked on `master` branch.
That's usually not a good thing :) . Always make a new branch from `master` when working on patches.

**Comment by tm9k1** [comment]
Also, it'd be better to replace the pre-existing JPG with an equal-or-better quality PNG if it's about transparency. If you have problems with updating this PR, drop a message (in this PR itself) with your issue.

**Comment by Ashish0804** [comment]
@brute4s99 Thanks for the feedback, the ones where i added .html or a github link were because it wasn't redirecting properly when running locally, adding .html or index.html fixes that problem

I'll be working on the changes as suggested :)

**Comment by Ashish0804** [comment]
Okay i have made all the suggested changes but now my website hosted on my repo won't function properly since "/" takes it to the root of ashish0804.github.io rather than ashish0804.github.io/opencode-mait-dark-theme.github/index.html

Also i have kept the transparent image as without it we will see an ugly white logo in the middle of the screen.

I suggest not to change the href from the explicit link so @ me if u want me to me change it too

**Comment by tm9k1** [comment]
I cannot test your fork currently, please fix it.

File: index.html (hunk 5 above)
Comment (tm9k1) [reply index.html:23]: it works without this change

**Comment by Ashish0804** [comment]
@brute4s99 Hi
wdym u can't test? is it the problem with "back to homepage" button?

**Comment by tm9k1** [comment]
nvm, It works. Thanks for the PR, @Ashish0804. 
Merging this.

**Comment by Ashish0804** [comment]
thanks :)

**Pull request merged**